.github/CHANGELOG.md

```diff
@@ -1,105 +1,188 @@
-# Release 0.13.0-dev
+# Release 0.13.0-dev (development release)
 
-### New features
+<h3>New features since last release</h3>
+
+* Adds initial support for the Xanadu cloud platform.
+  [(#101)](https://github.com/XanaduAI/strawberryfields/pull/101)
+  [(#148)](https://github.com/XanaduAI/strawberryfields/pull/148)
+  [(#294)](https://github.com/XanaduAI/strawberryfields/pull/294)
+
+  Jobs can now be submitted to the Xanadu cloud platform to be run
+  on supported hardware using the new `RemoteEngine`:
+
+  ```python
+  from strawberryfields import RemoteEngine
+  from strawberryfields import ops
+  from strawberryfields.utils import random_interferometer
+
+  # replace AUTHENTICATION_TOKEN with your Xanadu cloud token
+  eng = RemoteEngine("chip2", token="AUTHENTICATION_TOKEN")
+  prog = sf.Program(8)
+
+  U = random_interferometer(4)
+
+  with prog.context as q:
+      ops.S2gate(1.0) | (q[0], q[4])
+      ops.S2gate(1.0) | (q[1], q[5])
+      ops.S2gate(1.0) | (q[2], q[6])
+      ops.S2gate(1.0) | (q[3], q[7])
+
+      ops.Interferometer(U) | q[:4]
+      ops.Interferometer(U) | q[4:]
+
+  result = eng.run(prog)
+  ```
+
+  For more details, see the
+  [Xanadu cloud platform](https://strawberryfields.readthedocs.io/en/stable/introduction/remote.html)
+  tutorial.
+
+* Significantly speeds up the Fock backend of Strawberry Fields,
+  through a variety of changes:
+
+  - The Fock backend now uses The Walrus high performance implementations of
```

Comment:
Already merged into master?

Reply:
Yep, the was quite a large merge conflict in the changelog here, so it looks like a rewrite. (and it is)